Code/Tools/AssetProcessor/native/AssetManager/assetProcessorManager.h

```diff
@@ -290,7 +290,7 @@ namespace AssetProcessor
         virtual void AssessDeletedFile(QString filePath);
         void OnAssetScannerStatusChange(AssetProcessor::AssetScanningStatus status);
         void FinishAssetScan();
-        void OnJobStatusChanged(JobEntry jobEntry, JobStatus status);
+        void OnJobStatusChanged(JobEntry jobEntry, JobStatus status, int escalation, bool critical);
```

Comment:
Nit: Consider comments on public methods, unclear what "escalation means" here.

As a thought experiment, are there other statuses we may care about in the future ie should the bool by an enum?

Reply:
Added a comment on the signal (this is the slot).  I think bool is fine for the critical because this comes directly from `JobDetails` where critical is stored as a separate bool.